sdk/Sdk/Sdk.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <MinorProductVersion>15</MinorProductVersion>
-    <PatchProductVersion>1</PatchProductVersion>
+    <MinorProductVersion>16</MinorProductVersion>
```

Comment:
Shouldn't we also bump the generators project?

Reply:
There was no change to generators since our last worker.sdk release. #1968  updated only SDK project.